src/isolate.rs

```diff
@@ -195,6 +169,7 @@ impl Isolate {
     snapshot: libdeno::deno_buf,
     state: Arc<IsolateState>,
     dispatch: Dispatch,
+    permissions: Option<DenoPermissions>,
```

Comment:
Please add some triple slash docs here.

What does it mean when permissions is None? How is that different than having the Permission struct present but with all the fields set to false?